### PR TITLE
[15.0][MIG] sale_coupon_financial_risk: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43281

@chienandalu please review!

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Why the rename?

**Comment by stefan-tecnativa** [comment]
> Why the rename?

He had followed the logic used when separating certain coupon modules from their sales-related part. But now I realize that in this case, being directly related to sales, the renaming is not necessary.

File: sale_coupon_financial_risk/i18n/sale_coupon_financial_risk.pot [hunk 1]
```diff
@@ -1,19 +1,21 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-# 	* sale_coupon_financial_risk
+# 	* coupon_financial_risk
```
Comment (chienandalu) [inline sale_coupon_financial_risk/i18n/sale_coupon_financial_risk.pot]: Translation files are keeping the rename you made

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Remember to squash administrative commits into the generator one:

Selección_114

**Comment by stefan-tecnativa** [comment]
> Remember to squash administrative commits into the generator one:
> 
> Selección_114

Done, thanks for the advice.

**Pull request merged**